src/universal/components/Dashboard/DashSidebar.js

```diff
@@ -13,9 +13,9 @@ let styles = {};
 const DashSidebar = (props) => {
   const {activeArea, activeTeamId, dispatch} = props;
   const teamItems = props.user.memberships.map(m => ({
-    active: activeArea === 'team' && m.team.id === activeTeamId,
-    href: `/team/${m.team.id}`,
-    label: m.team.name
+    active: activeArea === 'team' && m.meeting.id === activeTeamId,
```

Comment:
I still hate this `activeArea` deal, since that is a function of the route we're in, but I figured this PR was far too large already.